projects/rml-datamart/setup.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+THISDIRFILE=$(dirname $0)
+cd $THISDIRFILE
+THISDIR=$(pwd -P)
+. "$HOMEBREWBARKLYDIR/lib/init.sh"
+
+# Start your script below and user any helpsers setup in lib/init.sh
+npm install grunt-cli
```

Comment:
Which directory will this run in? If not barkly/rml-datamart, then it won't be installed to the correct location. Could install it globally